hawkular-inventory-api/src/main/java/org/hawkular/inventory/api/configuration/Configuration.java

```diff
@@ -126,7 +137,8 @@ public String getProperty(Property property, String defaultValue) {
         Map<String, T> propsByName = overridableProperties.stream().collect(toMap(Property::getPropertyName,
                 identity()));
 
-        return implementationConfiguration.entrySet().stream().collect(toMap(Map.Entry::getKey, e -> {
+        return implementationConfiguration.entrySet().stream().filter(e -> !e.getKey().startsWith
+                (RestConfiguration.PROPERTY_PREFIX)).collect(toMap(Map.Entry::getKey, e -> {
```

Comment:
Am I right in saying that you exclude REST config from here because it's not really the "implementation configuration"?